pkgs/top-level/all-packages.nix

```diff
@@ -19560,6 +19560,10 @@ in
 
   iksemel = callPackage ../development/libraries/iksemel { };
 
+  imag = callPackage ../applications/office/imag {
```

Comment:
Nitpick: most CLI todo and contact list managers I know of are in `applications/misc`. Maybe this does belong to `applications/office` better, I'm not sure.

Reply:
I'm not sure either. It would certainly fit both, because its scope is PIM in general... I tend to prefer `applications/misc` though, as a "diary" for example is certainly not "office" :smile: